Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
+
+dockerizedBuildPipeline(
+  prepare: {
+    githubStatusUpdate('pending')
+  },
+  buildImageId: 'docker-all.repo.sonatype.com/alpine/helm',
+  buildAndTest: {
+    sh './build.sh'
+  },
+  skipVulnerabilityScan: true,
+  archiveArtifacts: 'docs/*',
```

Comment:
why are we archiving this entire folder? Seems like that will capture the entire catalog on each build...

Reply:
I did that to highlight the output of the job, which is the whole archive +1 the index.yaml that references them all. I could trim it down, but I'm not sure I know a meaningful subset.

Reply:
ideally only what is built, but these things turn out to be tiny anyhow so I'm fine with it